2014-07-22 04:00:46,914 Unit account update failure: bts.daft.1061992@faiyamrahman.com, beatthestreak1

2014-07-22 04:00:46,914 Unit account update failure: bts.daft.1061992@faiyamrahman.com, beatthestreak1

2014-07-22 04:00:46,914 Unit account update failure: bts.daft.1061992@faiyamrahman.com, beatthestreak1

2014-07-22 04:00:46,914 Unit account update failure: bts.daft.1061992@faiyamrahman.com, beatthestreak1

2014-07-22 04:00:46,914 Unit account update failure: bts.daft.1061992@faiyamrahman.com, beatthestreak1

2014-07-22 04:00:46,914 Unit account update failure: bts.daft.1061992@faiyamrahman.com, beatthestreak1

2014-07-22 04:00:46,914 Unit account update failure: bts.daft.1061992@faiyamrahman.com, beatthestreak1

2014-07-22 04:00:46,915     p1, p2: None, None

2014-07-22 04:00:46,915     p1, p2: None, None

2014-07-22 04:00:46,915     p1, p2: None, None

2014-07-22 04:00:46,915     p1, p2: None, None

2014-07-22 04:00:46,915     p1, p2: None, None

2014-07-22 04:00:46,915     p1, p2: None, None

2014-07-22 04:00:46,915     p1, p2: None, None

2014-07-22 04:00:46,915 Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
2014-07-22 04:00:46,915 Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
2014-07-22 04:00:46,915 Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
2014-07-22 04:00:46,915 Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
2014-07-22 04:00:46,915 Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
2014-07-22 04:00:46,915 Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
2014-07-22 04:00:46,915 Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Mike', 'Trout', 'laa') and ('Jose', 'Abreu', 'cws')" 
2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,548 Unit account update failure: bts.daft.330@faiyamrahman.com, beatthestreak1

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,549     p1, p2: None, None

2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:38:04,550 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012 Unit account update failure: bts.daft.37344@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,012     p1, p2: None, None

2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:06,013 Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
TimeoutException: Message: " with players ('Adrian', 'Beltre', 'tex') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,245 Unit account update failure: bts.daft.410@faiyamrahman.com, beatthestreak1

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246     p1, p2: None, None

2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:41:33,246 Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Mike', 'Trout', 'laa') and ('Miguel', 'Cabrera', 'det')" 
2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,494 Unit account update failure: bts.daft.668252@faiyamrahman.com, beatthestreak1

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495     p1, p2: None, None

2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
2014-07-22 04:54:39,495 Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
Traceback (most recent call last):
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 402, in choosePlayers
    eligiblePlayers=eligiblePlayers)
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/chooseplayers.py", line 172, in __distribute_eligible_players
    eligiblePlayers=kwargs['eligiblePlayers'])
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 21, in randDownRandPlayers
    doubleDown=random.choice((True, False)) )
  File "/home/faiyamrahman/programming/Python/beatthestreakBots/distributionfunctions.py", line 54, in staticDownRandPlayers
    raise type(e)(e.message + ' with players {} and {}'.format(p1, p2))
WebDriverException: Message: " with players ('Miguel', 'Cabrera', 'det') and ('Mike', 'Trout', 'laa')" 
